platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/properties/PropertiesLoader.java

```diff
@@ -142,6 +142,34 @@ public Properties loadProperties(String propertiesFile, ClassLoader classLoader)
     return properties;
   }
 
+  /**
+   * Will attempt to load properties from a file using the given classloader without replacing the
+   * system properties with their value. If that fails, several other methods will be tried until
+   * the properties file is located.
+   *
+   * <p>Note that loading the properties directly from the file system will replace the system
+   * properties with their corresponding values so if that method is used to load the properties,
+   * the system properties will be substituted.
+   *
+   * @param propertiesFile the resource name or the file path of the properties file
+   * @param classLoader the class loader with access to the properties file
+   * @return Properties deserialized from the specified file, or empty if the load failed
+   */
+  public Properties loadPropertiesWithoutSystemPropertySubstitution(
```

Comment:
This change isn't that trivial. It will require refactoring the entire class since the individual methods we call to try and load the properties all return a `java.util.Properties`. By the individual method I mean everything defined in [`PROPERTY_LOADING_STRATEGIES`](https://github.com/codice/ddf/blob/master/platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/properties/PropertiesLoader.java#L71). We can do this as a follow-on task with the migration change if necessary.